src/styles/blocks/card.scss

```diff
@@ -0,0 +1,44 @@
+@import "./blocks/variables.scss";
+
+.catalog__card {
+  margin-right: $card-gap;
+  margin-bottom: 46px;
+}
```

Comment:
It should be moved to `catalog.scss` file

Reply:
corrected